.github/workflows/build_and_scan_images.yaml

```
@@ -0,0 +1,28 @@
+name: Build and scan OCI images
+
+on:
+  schedule:
+  # every day at 1:00AM UTC
+  - cron: '0 1 * * *'
+
+jobs:
+  build-scan:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Build and scan for vulnerabilities
+        run: |
+          cd image-definitions/
+          ./install-tools.sh
+          ./setup.sh .
+          ./check-updates.sh || true
+          ./apply-patches.sh
+          ./build-scan.sh
+      - name: Send vulnerability records
+        run: |
+          cd image-definitions/
+          ./send-scan.sh ./trivy-reports
```

Just a suggestion.  Functionally the same, but a bit more github workflow-y
```suggestion
      - name: Send vulnerability records
        run: |
          ./send-scan.sh ./trivy-reports
        working-directory: image-definitions
```